### PR TITLE
[Stream] Web, iOS, Android Playback Documentation + Examples

### PR DESCRIPTION
**Pull request by dcpena**
- [x] Merge https://github.com/cloudflare/cloudflare-docs/pull/5585
- [x] Merge https://github.com/cloudflare/templates/pull/95

Video playback content including sections for iOS, Android, and web players. Part of Phase 2 for Stream docs (PCX-4369).

Also corrects `clientBandwidthHint` docs to be accurate (STREAM-4342)

### CODE REVIEW

**Comment by dcpena** [comment]
@irvinebroque I think this is in a good spot to start reviewing whenever you have a moment

File: content/stream/examples/hls-js.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+type: example
+summary: Example of hls.js playback with Cloudflare Stream
+tags:
+  - Playback
+pcx_content_type: configuration
+title: Video playback with hls.js
+layout: example
+---
+
+[Run and edit this code in your browser on Stackblitz.](https://stackblitz.com/edit/js-819kwb?file=index.js,package.json,index.html)
```
Comment (irvinebroque) [inline content/stream/examples/hls-js.md]: I'll take a pass at editing these with final links when we merge the final version into prod

File: content/stream/viewing-videos/using-own-player/_index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+title: Use your own player
+weight: 1
+layout: single
+---
+
+# Use your own player
+
+Cloudflare Stream is compatible with all video players that support HLS and DASH, which are standard formats for streaming media with broad support across all web browsers, mobile operating systems and media streaming devices.
+
+Refer to the guides below for more information.
```
Comment (irvinebroque) [inline content/stream/viewing-videos/using-own-player/_index.md]: Maybe "platform-specific guides"

File: content/stream/viewing-videos/using-own-player/_index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+* [Web](/stream/viewing-videos/using-own-player/web/)
+* [iOS (AVPlayer)](/stream/viewing-videos/using-own-player/ios/)
+* [Android (ExoPlayer)](/stream/viewing-videos/using-own-player/android/)
+
+## Fetch HLS and Dash manifests
+
+### URL 
+
+Each video and live stream has its own unique HLS and DASH manifest. You can access the manifest by replacing `<UID>` with the UID of your video or live input, and replacing `<CODE>` with your unique customer code, in the URLs below:
```
Comment (irvinebroque) [inline content/stream/viewing-videos/using-own-player/_index.md:21]: We can follow up and update this with something that explains where you can find this customer code

File: content/stream/viewing-videos/using-own-player/android.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+## Examples
+
+Refer to the [Android example](/stream/examples/android/) to view a working example of Cloudflare Stream and ExoPlayer.
+
+## Use ExoPlayer and HLS
+
+Refer to the code example below for a basic implementation of ExoPlayer using Cloudflare Stream's Manifest URL.
+
+
+```
```
Comment (irvinebroque) [inline content/stream/viewing-videos/using-own-player/android.md]: This should be `kotlin`

File: content/stream/viewing-videos/using-own-player/android.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+- A customizable player
+- Support for playlists
+- Support for DASH and SmoothStreaming
+- Support for HLS
+
+{{<render file="_prereqs.md">}}
+
+## Examples
+
+Refer to the [Android example](/stream/examples/android/) to view a working example of Cloudflare Stream and ExoPlayer.
```
Comment (irvinebroque) [inline content/stream/viewing-videos/using-own-player/android.md]: "to download and run an example Android app that uses ExoPlayer to play video content from Cloudflare Stream"

Comment (irvinebroque) [reply content/stream/viewing-videos/using-own-player/android.md]: We can highlight here that this path lets you _run_ something, not just view code snippets.

File: content/stream/viewing-videos/using-own-player/ios.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+Cloudflare Stream can be used with Apple’s built-in [AVPlayer](https://developer.apple.com/documentation/avfoundation/avplayer), which is integrated into iOS and provides a variety of features by default, including but not limited to:
+
+- Picture-in-picture
+- Lock screen controls
+- Gestures
+- Theming
+- Accessibility
+- Foreground/background tasks
+- Concurrency management with other device sensors and system notifications
+- SwiftUI compatibility
```
Comment (irvinebroque) [inline content/stream/viewing-videos/using-own-player/ios.md]: I'll edit, but for now let's get this merged into the other branch

**Review by irvinebroque — APPROVED** [review]
_

**Pull request merged**